packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -482,7 +482,7 @@
             }
         }
         &__list {
-            overflow-y: scroll;
```

Comment:
what happened here

Reply:
Someone broke the component by adding this css rule and QA didn't test it 😛

Reply:
What exactly was broken? This css was added in Turbos PR

Reply:
I'm interested, because person who broke might be me))))

Reply:
<img width="242" alt="image" src="https://github.com/binary-com/deriv-app/assets/71861519/92d3a9e2-3772-4907-b8b6-2075ec7e3b29">
This is on production @kate-deriv. There's a forced scrollbar where it shouldn't be. I think we shouldn't force a scrollbar anywhere, always make it `auto`.